Bootstraps/Symfony.php

```diff
@@ -49,16 +49,24 @@ public function getApplication()
         $appAutoLoader = './app/autoload.php';
         if (file_exists($appAutoLoader)) {
             require $appAutoLoader;
-        } else {
+        } elseif (file_exists('./vendor/autoload.php')) {
             require './vendor/autoload.php';
         }
+        if(class_exists('\AppKernel')) {
+            $app = new \AppKernel($this->appenv, $this->debug);
+        } else {
+            if(file_exists('./app/bootstrap.php.cache') && file_exists('./app/AppKernel.php')) {
```

Comment:
I got the pre 2.5 style of installation. Ex.  cache is in app/cache instead of var/cache